### ISSUE TITLE
`JediDebugInfo` fails trying to get sys_path

### ISSUE THREAD

**Issue by nikvdp** [davidhalter/jedi-vim#813]
### Issue

Running `:JediDebugInfo` leads to an error message about sys_path.  

```
#### Jedi-vim debug information
Using Python version: 3
 - sys.version: `3.5.2 (default, Nov 23 2017, 16:37:01), [GCC 5.4.0 20160609]`
 - site module: `/usr/lib/python3.5/site.py`
Jedi path: `/home/nik/.vim/bundle/jedi-vim/pythonx/jedi/jedi/__init__.py`
 - version: 0.12.0
 - sys_path:
Error detected while processing function jedi#debug_info:
line   28:
Traceback (most recent call last):
  File "/home/nik/.vim/bundle/jedi-vim/pythonx/jedi_vim_debug.py", line 37, in display_debug_info
    sys_path = script_evaluator.project.sys_path
AttributeError: 'Project' object has no attribute 'sys_path'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nik/.vim/bundle/jedi-vim/pythonx/jedi_vim_debug.py", line 39, in display_debug_info
    sys_path = script_evaluator.sys_path
AttributeError: 'Evaluator' object has no attribute 'sys_path'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/nik/.vim/bundle/jedi-vim/pythonx/jedi_vim_debug.py", line 44, in display_debug_info
    "{0!r})".format(e))
  File "/home/nik/.vim/bundle/jedi-vim/pythonx/jedi_vim_debug.py", line 8, in echo
    vim.command('echo {0}'.format(msg))
vim.error: Vim(echo):E117: Unknown function: AttributeError
E171: Missing :endif


```

### Steps to reproduce

1. `git clone --recursive https://github.com/davidhalter/jedi-vim.git ~/.vim/bundle/jedi-vim`
2. copy `minimal.vimrc` (contents below) into `~/.vim/bundle/jedi-vim`

   ```vim
   set nocompatible

   let script_dir = fnamemodify(expand('<sfile>'), ':h')
   let &runtimepath .= ','.script_dir.','.script_dir.'/after'

   " Put your config changes here.
   " let g:jedi#show_call_signatures=1

   syntax on
   filetype plugin indent on
   ```
3. run `vim -u ~/.vim/bundle/jedi-vim/minimal.vimrc`
4. run `:verbose JediDebugInfo` inside vim.

The error is the same in vim 7.4 and neovim 0.2.2, and happens with both ubuntu's python 3.5, anaconda's python 3.6, on linux and on OS X.

### Output of “:verbose JediDebugInfo”

```
#### Jedi-vim debug information
Using Python version: 3
 - sys.version: `3.5.2 (default, Nov 23 2017, 16:37:01), [GCC 5.4.0 20160609]`
 - site module: `/usr/lib/python3.5/site.py`
Jedi path: `/home/nik/.vim/bundle/jedi-vim/pythonx/jedi/jedi/__init__.py`
 - version: 0.12.0
 - sys_path:
Error detected while processing function jedi#debug_info:
line   28:
Traceback (most recent call last):
  File "/home/nik/.vim/bundle/jedi-vim/pythonx/jedi_vim_debug.py", line 37, in display_debug_info
    sys_path = script_evaluator.project.sys_path
AttributeError: 'Project' object has no attribute 'sys_path'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/nik/.vim/bundle/jedi-vim/pythonx/jedi_vim_debug.py", line 39, in display_debug_info
    sys_path = script_evaluator.sys_path
AttributeError: 'Evaluator' object has no attribute 'sys_path'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/nik/.vim/bundle/jedi-vim/pythonx/jedi_vim_debug.py", line 44, in display_debug_info
    "{0!r})".format(e))
  File "/home/nik/.vim/bundle/jedi-vim/pythonx/jedi_vim_debug.py", line 8, in echo
    vim.command('echo {0}'.format(msg))
vim.error: Vim(echo):E117: Unknown function: AttributeError
E171: Missing :endif


```

**Comment by davidhalter** [comment]
Please upgrade Jedi. It's probably very old.

**Comment by blueyed** [comment]
No, it is currently broken.
The fix is included in https://github.com/davidhalter/jedi-vim/pull/806.

**Comment by davidhalter** [comment]
Oh shit. I didn't realize that it still wasn't merged. My fault.

**Comment by davidhalter** [comment]
Still closing, please wait for #806. I really hope we can merge soon. Sorry for the inconvenience.

**Comment by Mrshnab** [comment]
https://travis-ci.org/ankane/ip_anonymizer.svg?branch=master

**Comment by davidhalter** [comment]
I merged #806 now. It should work again.
